src/styles/blocks/catalog.scss

```diff
@@ -17,9 +17,16 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    transition: transform $transition-duration-value;
+    transition-property: transform, color;
```

Comment:
you can use transition in this way
<img width="637" alt="image" src="https://github.com/mate-academy/layout_catalog_hovers/assets/102411130/4979c271-db75-40e8-bb27-e2e174fdaf2f">

Reply:
https://www.w3schools.com/css/css3_transitions.asp